### PR TITLE
Add percentile getter methods to CILViewerBase and CILViewer

### LINKED ISSUE #335: Add getVolumeColorPercentiles, getScalarOpacityPercentiles methods etc.
Currently we have methods:
https://github.com/vais-ral/CILViewer/blob/d1d29f9f3e7d3db180104f9f2446452a50f33bf8/Wrappers/Python/ccpi/viewer/CILViewer.py#L762

https://github.com/vais-ral/CILViewer/blob/d1d29f9f3e7d3db180104f9f2446452a50f33bf8/Wrappers/Python/ccpi/viewer/CILViewer.py#L737

and:
https://github.com/vais-ral/CILViewer/blob/d1d29f9f3e7d3db180104f9f2446452a50f33bf8/Wrappers/Python/ccpi/viewer/CILViewer.py#L752

We have getters for the values but not for the percentages. E.g.
https://github.com/vais-ral/CILViewer/blob/d1d29f9f3e7d3db180104f9f2446452a50f33bf8/Wrappers/Python/ccpi/viewer/CILViewer.py#L839

But we don't have a method to get the percentages. We should add methods such as:

`getVolumeColorPercentiles()`

### PR DESCRIPTION
**Pull request by lauramurgatroyd**
Closes #335 
Adds unit tests for the new methods

### CODE REVIEW

**Comment by lauramurgatroyd** [comment]
Added line for sipping cilviewer test in gh-a for docker

File: .github/workflows/docker_build_test_publish.yml [hunk 1]
```diff
@@ -33,5 +33,5 @@ jobs:
       - name: Run docker container with tests
         shell: bash -l {0}
         run: |
-          docker run --rm --entrypoint /bin/bash -v /home/runner/work/CILViewer/CILViewer:/root/source_code  cil-viewer -c "source ./mambaforge/etc/profile.d/conda.sh && conda activate cilviewer_webapp && conda install cil-data pytest -c ccpi && python -m pytest /root/source_code/Wrappers/Python -k 'not test_version and not test_cli_resample'"
+          docker run --rm --entrypoint /bin/bash -v /home/runner/work/CILViewer/CILViewer:/root/source_code  cil-viewer -c "source ./mambaforge/etc/profile.d/conda.sh && conda activate cilviewer_webapp && conda install cil-data pytest -c ccpi && python -m pytest /root/source_code/Wrappers/Python -k 'not test_version and not test_cli_resample and not test_CILViewerBase'"
```
Comment (lauramurgatroyd) [inline .github/workflows/docker_build_test_publish.yml:36]:
```suggestion
          docker run --rm --entrypoint /bin/bash -v /home/runner/work/CILViewer/CILViewer:/root/source_code  cil-viewer -c "source ./mambaforge/etc/profile.d/conda.sh && conda activate cilviewer_webapp && conda install cil-data pytest -c ccpi && python -m pytest /root/source_code/Wrappers/Python -k 'not test_version and not test_cli_resample and not test_CILViewerBase and not testCILViewer3D'"
```

File: .github/workflows/docker_build_test_publish.yml [hunk 2]
```diff
@@ -33,5 +33,5 @@ jobs:
       - name: Run docker container with tests
         shell: bash -l {0}
         run: |
-          docker run --rm --entrypoint /bin/bash -v /home/runner/work/CILViewer/CILViewer:/root/source_code  cil-viewer -c "source ./mambaforge/etc/profile.d/conda.sh && conda activate cilviewer_webapp && conda install cil-data pytest -c ccpi && python -m pytest /root/source_code/Wrappers/Python -k 'not test_version and not test_cli_resample'"
+          docker run --rm --entrypoint /bin/bash -v /home/runner/work/CILViewer/CILViewer:/root/source_code  cil-viewer -c "source ./mambaforge/etc/profile.d/conda.sh && conda activate cilviewer_webapp && conda install cil-data pytest -c ccpi && python -m pytest /root/source_code/Wrappers/Python -k 'not test_version and not test_cli_resample and not test_CILViewerBase and not testCILViewer3D'"
```
Comment (lauramurgatroyd) [inline .github/workflows/docker_build_test_publish.yml:36]:
```suggestion
          docker run --rm --entrypoint /bin/bash -v /home/runner/work/CILViewer/CILViewer:/root/source_code  cil-viewer -c "source ./mambaforge/etc/profile.d/conda.sh && conda activate cilviewer_webapp && conda install cil-data pytest -c ccpi && python -m pytest /root/source_code/Wrappers/Python -k 'not test_version and not test_cli_resample and not test_CILViewerBase and not test_CILViewer3D'"
```

File: Wrappers/Python/test/test_CILViewerBase.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
+#   Copyright 2022 STFC, United Kingdom Research and Innovation
+#
+#   Author 2022 Samuel Jones
```
Comment (lauramurgatroyd) [inline Wrappers/Python/test/test_CILViewerBase.py:3]:
```suggestion
#   Author 2023 Laura Murgatroyd
```

File: Wrappers/Python/test/test_CILViewer3D.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
+#   Copyright 2022 STFC, United Kingdom Research and Innovation
+#
+#   Author 2022 Samuel Jones
```
Comment (lauramurgatroyd) [inline Wrappers/Python/test/test_CILViewer3D.py:3]:
```suggestion
#   Author 2023 Laura Murgatroyd
```

File: Wrappers/Python/test/test_CILViewer3D.py [hunk 5]
```diff
@@ -0,0 +1,82 @@
+#   Copyright 2022 STFC, United Kingdom Research and Innovation
```
Comment (lauramurgatroyd) [inline Wrappers/Python/test/test_CILViewer3D.py:1]:
```suggestion
#   Copyright 2023 STFC, United Kingdom Research and Innovation
```

File: Wrappers/Python/test/test_CILViewerBase.py [hunk 6]
```diff
@@ -0,0 +1,54 @@
+#   Copyright 2022 STFC, United Kingdom Research and Innovation
```
Comment (lauramurgatroyd) [inline Wrappers/Python/test/test_CILViewerBase.py:1]:
```suggestion
#   Copyright 2023 STFC, United Kingdom Research and Innovation
```

**Review by paskino — APPROVED** [review]

**Pull request merged**